common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1369,6 +1369,21 @@ public static String computeMaxHostBrokerProtocol() {
          */
         public static final String REMOVE_BROKER_ACCOUNT_SUCCEEDED = "remove_broker_account_succeeded";
 
+        /**
+         * Boolean to return when a transfer token is successfully saved.
+         */
+        public static final String SAVE_TRANSFER_TOKEN_SUCCEEDED = "save_transfer_token_succeeded";
+
+        /**
+         * The Bundle key name of transfer token.
+         */
+        public static final String TRANSFER_TOKEN = "transfer_token_value";
```

Comment:
So this is the key or the value? The actual content seems to call it value but javadoc says key

Reply:
This is bundle key only.  We are passing transfer token value using this key. Let me just call it TRANSFER_TOKEN_BUNDLE_KEY to avoid any confusion.